### DISCUSSION TITLE
Any help would be appreciated

### DISCUSSION THREAD

**Discussion by MsDims23** [community/community#64914] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![IMG_2886](https://github.com/community/community/assets/127474752/87fbd2e6-7364-4279-a14f-d030813348b0)
I had this icon show up on my laptop. Does anyone know what this is? I can’t find anything about it. My fiancé id hacking all my accounts phone and laptop, he denies it but I know is he. I don’t know much about computers and would love some help or just an answer. He switch most my accounts into his name so I can’t even get real answers from the companies

**Comment by advayc** [comment]
I believe that button is the task view button. Task View is a feature in Windows that allows you to manage your opened applications and virtual desktops, as well as a history of recently opened files. Task View is a powerful tool for organizing your work and increasing productivity.
